index.html

```diff
@@ -267,23 +267,20 @@
 </pre>
 
 <p>
-Each <code>Document</code> has:
+Each <code>Window</code> has:
 </p>
 <ul>
-<li>a <dfn>list of idle request callbacks</dfn>, which MUST be initially empty,</li>
-<li>a <dfn>list of runnable idle callbacks</dfn>, which MUST be initially empty,</li>
-<li>and an <dfn>idle callback identifier</dfn>, which is a number which MUST initially be zero.</li>
+<li>A <dfn>list of idle request callbacks</dfn>. The list MUST be initially empty and each entry in this list is identified by a number, which MUST be unique within the list for the lifetime of the <code>Window</code> object.</li>
```

Comment:
Not really. It's either a list or a map. A list with each entry identified by a number that is unique within the list is just a really awkward way of writing a map.

It'd be better to call it a map, and use the correct terminology (entries, keys, values). You can see some examples of map prose in e.g. https://html.spec.whatwg.org/multipage/webappapis.html#fetch-a-single-module-script which manipulates the module map.

This is fairly orthogonal to your PR though so it's fine to tackle it in a separate issue.